omnipod-dash/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/dash/OmnipodDashPumpPlugin.kt

```diff
@@ -340,20 +340,23 @@ class OmnipodDashPumpPlugin @Inject constructor(
                     aapsLogger.info(LTag.PUMP, "syncBolusWithPumpId on CANCEL_BOLUS returned: $sync")
                 }
             }
-            showNotification(
-                Notification.OMNIPOD_POD_FAULT,
-                podStateManager.alarmType.toString(),
-                Notification.URGENT,
-                R.raw.boluserror
-            )
-            if (!podStateManager.alarmSynced) {
-                pumpSync.insertAnnouncement(
-                    error = podStateManager.alarmType?.toString() ?: "Unknown pod failure",
-                    pumpId = Random.Default.nextLong(),
-                    pumpType = PumpType.OMNIPOD_DASH,
-                    pumpSerial = serialNumber()
+
+            podStateManager.alarmType?.let {
+                showNotification(
+                    Notification.OMNIPOD_POD_FAULT,
+                    it.toString(),
+                    Notification.URGENT,
+                    R.raw.boluserror
```

Comment:
There is still the issue that this sound does not stop playing when the notification is dismissed.